packages/types/src/codec/Struct.ts

```diff
@@ -43,30 +46,54 @@ export default class Struct <
       }, {} as E);
   }
 
-  static decode <S, V, T> (Types: S, value: V | Array<any>, isTuple: boolean): T {
+  static decode<S, V, T> (Types: S, value: V | Array<any> | AnyU8a, isTuple: boolean): T {
```

Comment:
Erk, the isTuple is a hold-over, I don't believe is is used anymore, Tuples do their own thing. (It was pre-changes when stuff still came in as `Array<number>`)